charts/vsphere-cpi/v1.0.0/README.md

```diff
@@ -0,0 +1,50 @@
+# vSphere Cloud Provider Interface (CPI)
+
+[vSphere Cloud Provider Interface (CPI)](https://github.com/kubernetes/cloud-provider-vsphere) is responsible for running all the platform specific control loops that were previously run in core Kubernetes components like the KCM and the kubelet, but have been moved out-of-tree to allow cloud and infrastructure providers to implement integrations that can be developed, built and released independent of Kubernetes core. The official documentation and tutorials can be found [here](https://vsphere-csi-driver.sigs.k8s.io/driver-deployment/prerequisites.html).
+
+## Prerequisites
+
+- vSphere 6.7 U3+
+- Kubernetes v1.14+
+- A Secret on your Kubernetes cluster that contains vSphere credentials (Refer to `README` or `Detailed Descriptions`)
+
+## Installation
+
+This chart requires a secret in your Kubernetes cluster that contains the server URL and credentials to connect to the vCenter. You can have the chart generate it for you, or create it yourself and provide the name of the secret during installation. (<span style="color:orange">Warning</span>: When the option to generate the secret is enabled, the credentials are visible in the API to authorized users. If you create the secret yourself they will not be visible). You can create it in one of the following ways:
+### <B>Option 1</b>: Create a secret using the Rancher UI
+Go to your cluster's project (Same project you will be installing the chart) > Resources > Secrets > Add Secret.
+```yaml
+# Example of data required in the secret
+<host-1>.username: <username>
+<host-1>.password: <password>
+```
+
+### <B>Option 2</b>: Create a secret using kubectl
+Replace placeholders with actual values, and execute the following:
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+    name: <secret-name>
+    namespace: <charts-namespace>
+data:
+    <host-1>.username: <base64encoded-username>
+    <host-1>.password: <base64encoded-password>
+EOF
```

Comment:
I think we should add a link to https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl/ for more information